src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -304,6 +294,18 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
                     el.innerHTML = processedValue;
 
+                    if (domUtils.isBodyElement(el)) {
+                        var shadowUIRoot = this.shadowUI.select('.' + this.shadowUI.ROOT_CLASS, el)[0];
```

Comment:
This code will not create root if it doesn't exist.
The `shadowUI.getRoot` will create a root.
So, it is not a big problem and I will use `shadowUI.getRoot`.